schema/partner_shows.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import gravity from '../lib/loaders/gravity';
+import PartnerShowSorts from './sorts/partner_show_sorts';
+import PartnerShow from './partner_show';
+import {
+  GraphQLString,
+  GraphQLList,
+  GraphQLInt,
+  GraphQLBoolean,
+  GraphQLEnumType
+} from 'graphql'
+
+let PartnerShows = {
+  type: new GraphQLList(PartnerShow.type),
+  description: 'A list of PartnerShows',
+  args: {
+    size: {
+      type: GraphQLInt
+    },
+    sort: PartnerShowSorts,
+    status: {
+      type: GraphQLString
```

Comment:
This can be a sep PR but this should probably be an enum too? (The nice thing being you get autocompletion/validation when it's an enum.)